### DISCUSSION TITLE
Option to hide the tests by default

### DISCUSSION THREAD

**Discussion by fcFn** [typehero/typehero#1224] [General]
Hello! 

Just went through the first challenge. First of all, great job! Ability to add your vim config? Wow!! 🤯 And that Rust joke also made me chuckle 😊

Unfortunately it wasn't much of a challenge since you immediately notice the answer in the test case source which spoils the fun. Should I perhaps open a feature request issue? There was PR #1089 that made the test case panel visible by default.

**Comment by bautistaaa** [comment]
first day did that on purpose. they won’t be giving you the answer anymore
